package.json

```
@@ -220,7 +220,7 @@
     "*.ts": "eslint --cache --cache-strategy content --fix"
   },
   "engines": {
-    "node": "~18",
+    "node": "^18.8.0",
```

```suggestion
    "node": "^18.18.0",
```
I think maybe a typo?

---

Yep, typo. Thanks for pointing that out, fixed in [06334c7](https://github.com/bitwarden/clients/pull/8390/commits/06334c7b6a0dc2b128a5c0348fff96130b07c906)